lib/haml/helpers.rb

```diff
@@ -607,9 +607,12 @@ def haml_tag_if(condition, *tag)
     # @param text [String] The string to sanitize
     # @return [String] The sanitized string
     def html_escape(text)
-      ERB::Util.html_escape(text)
```

Comment:
ActiveSupport monkey-patches this method for `html_safe?`

Reply:
Definitely like seeing the `CGI` version vs leaning on ActiveSupport